### ISSUE TITLE
Shortcuts editor error

### ISSUE THREAD

**Issue by AlbertoFabbri93** [crow-translate/crow-translate#468]
**Bug description**
I have a Swedish keyboard and I want to set the shortcut to Ctrl+½ but when I press those keys in the shortcut editor field I get Ctrl+\ instead and that obviously does not work.

**Steps to reproduce**
1. Go to 'Settings' --> 'Shortcut'
2. Click on 'Translate Selected Text'
3. Go to the input field called 'Key Sequence'
4. Press Ctrl+½ with the Swedish Keyboard layout selected from settings

**Expected behavior**
Ctrl+½ appears in the input box.

**Screenshots**
![Screenshot 2022-08-18 200236](https://user-images.githubusercontent.com/51910894/185463221-21aab899-c194-4737-875a-1f846392df69.png)

**Environment**
- OS: Windows 11
- Crow Translate version: 2.9.12

**Comment by Shatur** [comment]
I think it's an upstream issue. Press detection is done on Qt side.

**Comment by AlbertoFabbri93** [comment]
Do you know where I can report this issue to the Qt framework?

**Comment by Shatur** [comment]
I think you can report it here: https://bugreports.qt.io/secure/Dashboard.jspa

**Comment by AlbertoFabbri93** [comment]
It looks like there are already a ton of bugs opened about keyboard shortcuts. I do not feel like opening a new one because I do not know the implementation details of Qt in Crow-Translate.
Where does Crow-Translate save the settings? Is it possible to modify the shortcuts manually?

**Comment by Shatur** [comment]
> It looks like there are already a ton of bugs opened about keyboard shortcuts.

Yeah, this keyboard widget ([QKeySequenceEdit](https://doc.qt.io/qt-6/qkeysequenceedit.html)) is buggy. See, for example, #103. Usually Qt provides a very high quality platform integration, but this particular part probably tricky.

>  Where does Crow-Translate save the settings? Is it possible to modify the shortcuts manually?

This could work. Try to edit `HKEY_CURRENT_USER\Software\Crow translate\Crow translate` in registry (see [this](https://doc.qt.io/qt-5/qsettings.html#locations-where-application-settings-are-stored) for details).
And if you use portable mode, then the settings are stored next to the executable file.

**Comment by AlbertoFabbri93** [comment]
> Try to edit `HKEY_CURRENT_USER\Software\Crow translate\Crow translate` in registry (see [this](https://doc.qt.io/qt-5/qsettings.html#locations-where-application-settings-are-stored) for details).

This worked! Thank you very much!
